pkg/operation/botanist/etcd.go

```diff
@@ -51,14 +51,23 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 		return nil, err
 	}
 
+	var replicas *int32
+	if !b.Shoot.HibernationEnabled {
+		if gardenletfeatures.FeatureGate.Enabled(features.HAControlPlanes) && metav1.HasAnnotation(b.Shoot.GetInfo().ObjectMeta, v1beta1constants.ShootAlphaControlPlaneHighAvailability) {
+			replicas = pointer.Int32(3)
+		} else {
+			replicas = pointer.Int32(1)
+		}
```

Comment:
Move this into a function that you can reuse below in the `ScaleUpETCD` function?